Makefile

```diff
@@ -220,12 +220,15 @@ cluster-logging-cleanup: cluster-logging-operator-uninstall cluster-logging-cata
 # builds an operator-registry image containing the cluster-logging operator
 cluster-logging-catalog-build: .make/cluster-logging-catalog-build
 .make/cluster-logging-catalog-build: $(shell find olm_deploy -type f)
+	LOCAL_IMAGE_CLUSTER_LOGGING_OPERATOR_REGISTRY=$(REGISTRY_PUBLIC)/openshift/cluster-logging-operator-registry \
```

Comment:
shouldnt this be full swap of the image URL

Reply:
sorry, i didn't get you.

Reply:
should this be something like `LOCAL_IMAGE_CLUSTER_LOGGING_OPERATOR_REGISTRY=${SOMEOTHERVAR_TO_A_FULL_URL}`

Reply:
this line i copied from https://github.com/openshift/cluster-logging-operator/pull/1064/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R151

Reply:
and just moved it below, i suspected that the environment variables are not getting passed from one make target to another

Reply:
Make targets don't _pass_ env vars; assignments of the form `export FOO=BAR` modify the env of the make process, and that env is inherited by the shells that execute the commands in a target. The trick is the _order of evaluation_ of assignments and targets. That is black magic that no mere mortal can comprehend.  My latest change  https://github.com/openshift/cluster-logging-operator/pull/1070 should avoid this ordering madness - please try a rebase.